AntennaTracker/Tracker.cpp

```diff
@@ -23,37 +23,38 @@
 #include "version.h"
 #undef FORCE_VERSION_H_INCLUDE
 
-#define SCHED_TASK(func, _interval_ticks, _max_time_micros) SCHED_TASK_CLASS(Tracker, &tracker, func, _interval_ticks, _max_time_micros)
+#define SCHED_TASK(func, _interval_ticks, _max_time_micros, _prio) SCHED_TASK_CLASS(Tracker, &tracker, func, _interval_ticks, _max_time_micros, _prio)
 
 /*
   scheduler table - all regular tasks apart from the fast_loop()
   should be listed here, along with how often they should be called
   (in 20ms units) and the maximum time they are expected to take (in
   microseconds)
  */
-const AP_Scheduler::Task Tracker::scheduler_tasks[] = {
```

Comment:
tiny thing but I prefer the curly bracket on the previous line..

Reply:
emacs was sending me nuts with auto-formatting. I'll revert this next iteration.

Reply:
Fixed!